File: backend/ModuMessenger/src/main/java/com/example/modumessenger/member/dto/RequestMemberDto.java
package com.example.modumessenger.member.dto;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.io.Serializable;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class RequestMemberDto implements Serializable {

    private String userId;
    private String auth;
    private String email;
    private String username;
    private String statusMessage;
    private String profileImage;
}

File: backend/member-service/src/main/kotlin/com/example/memberservice/member/dto/RequestMemberDto.kt
package com.example.memberservice.member.dto

import com.example.memberservice.member.entity.Role
import java.io.Serializable

data class RequestMemberDto(
    var id: Long? = null,
    var userId: String? = null,
    var auth: String? = null,
    var role: Role? = null,
    var email: String? = null,
    var username: String? = null,
    var statusMessage: String? = null,
    var profileImage: String? = null,
    var wallpaperImage: String? = null,
) : Serializable
